### ISSUE TITLE
Command parameters with comments are not recognized

### ISSUE THREAD

**Issue by thyseus** [robersonfaria/laravel-database-schedule#2]
Hi,

at first - thanks a lot for this great repository. 

Unfortunately Commands that have Arguments with a comment are not recognized by `laravel-database-schedule`. Take this Example of the official `telescope` module:

```php
class PruneCommand extends Command
{
    /**
     * The name and signature of the console command.
     *
     * @var string
     */
    protected $signature = 'telescope:prune {--hours=24 : The number of hours to retain Telescope data}';
```
The argument `--hours` is not recognized and therefore can not be set:

![grafik](https://user-images.githubusercontent.com/654271/111447775-0ea63e80-870e-11eb-888a-70a3e80ae447.png)

The parameters are collected in `src/Http/Services/CommandService.php` but unfortunately i can not grasp why 
`$command->getDefinition()->getArguments())` in line 31 does not retrieve commented arguments. Possibly this is a laravel issue ?

Besides of that, how about an switch to enter a completely custom command as text field in addition to the dropdown for some special cases?

**Comment by robersonfaria** [comment]
Hi @thyseus , thanks for the comment.

I will check this parameter issue with comment and soon I will give you a return, it was a situation that I had not yet caught.

About the custom command, it is a good idea, would you like to send a PR adding this functionality? I would be really grateful.

Thanks

**Comment by thyseus** [comment]
Yes. Will do this today or tomorrow - i already have a "proof of concept" :) My first time with vue.js but it works great ! Stay tuned...

**Comment by robersonfaria** [comment]
I just released version 1.0.13 which corrects the problem with the options task parameters.
The problem was not exactly in the comment of the parameter.

I already took the opportunity to test some other things that were pending and fix other bugs.

I look forward to your contribution, thank you very much.

**Comment by thyseus** [comment]
https://github.com/robersonfaria/laravel-database-schedule/pull/3 here is my first attempt. Thanks a lot for reviewing :) Let me know what i need to improve to get this merged.
